### ISSUE TITLE
Crash when loading almost 4 GB big gw.dat file

### ISSUE THREAD

**Issue by Miniputs** [Jonathan-Greve/GuildWarsMapBrowser#15]
version 5.5
I copied old gw.dat file from old laptop to desktop PC and got crash when GWMB tries to load the almost 4 GB big file. (3,86 GB (4.150.691.840 bytes))
GW itself loads and plays the big file fine. GWMB also crash when start with small dat file and load the big file for compare, but it seems the crash happens later in the loading process.
[CrashDump.dmp](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/files/14002229/CrashDump.dmp)

**Comment by Jonathan-Greve** [comment]
Thanks for providing the CrashDump file. I've located the issue and fixed it in [b9768ff](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/commit/b9768ff4fc9eb6cb2219e5edfb2e3491d1d4c7d2).
The fix will be included in the next release which should be up within a few days.

**Comment by Miniputs** [comment]
Git cloned today and still got crash with both Release and Debug build. Now it seems to load more before crash (56%).
Got no CrashDump file, but I may need to add something like a parameter before I build.
Used latest commit: 12eba473c4994445133e4e07e2157eeebb367ff8

**Comment by Jonathan-Greve** [comment]
Ah damn, had hoped [b9768ff](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/commit/b9768ff4fc9eb6cb2219e5edfb2e3491d1d4c7d2) was the fix. If you ran the program from within Visual Studio with the debugger it won't create a crashdump file. Try going to the Debug folder instead and run the program (GuildWarsMapBrowser.exe) from there outside Visual Studio, then it should create a crashdump file. If you get a crashdump file please also provide the GuildWarsMapBrowser.pdb file.

You could also upload the .dat somewhere and I could download it to take a look. Although I understand it can be difficult to upload 4GB.

My guess is one of the files in the .dat is corrupted and the decompress method is crashing the program.

**Comment by Miniputs** [comment]
Used the latest commit and it no longer crashes. 
I got no crashdump file from the https://github.com/Jonathan-Greve/GuildWarsMapBrowser/commit/b9768ff4fc9eb6cb2219e5edfb2e3491d1d4c7d2 fix even run "standalone".
I could upload the dat file somewhere, but not sure Arenanet will be happy about it, even with password.
I actually got 4 same errors from the previous fix and progress to 72% for last error
![GWMB-error](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/assets/53795477/153a4b9c-8bad-4e30-85dd-b2d5c0ccc7c6)
